project/app/urls.py

```diff
@@ -8,8 +8,6 @@
     path("profile", views.ProfileView.as_view(), name="profile"),
     path("admin", views.AdminAccessView.as_view(), name="admin_login"),
     path("updateProfile/<str:pk>", views.UpdateProfileView.as_view(), name="admin_profile_update"),
-    path("admin-createuser", views.AdminAddUserView.as_view(), name="admin-create-user"),
-    path("manager-createuser", views.ManagerAddUserView.as_view(), name="manager-createuser"),
-    path("admin-createuserprofile", views.AdminAddProfile.as_view(), name="admin-create-userprofile"),
-    path("manager-createuserprofile", views.ManagerAddProfile.as_view(), name="manager-createuserprofile"),
-]
+    path("createuser", views.AddUserView.as_view(), name="admin-create-user"),
```

Comment:
Write the endpoint by using the standard name.
example:  'user' - POST method represent -> Create.